pkg/apimachinery/announced/group_factory.go

```diff
@@ -156,6 +156,15 @@ func (gmf *GroupMetaFactory) Register(m *registered.APIRegistrationManager) erro
 	prioritizedVersions = append(prioritizedVersions, unprioritizedVersions...)
 	m.RegisterVersions(prioritizedVersions)
 	gmf.prioritizedVersionList = prioritizedVersions
+
+	// Register empty group. Will be filled with proper values in Enable()
+	groupMeta := &apimachinery.GroupMeta{
```

Comment:
We have a problem which I try to fix here (WIP): if you disable api groups, they are not registered anymore. You cannot create a clientset because it doesn't find the group in the registration manager.